crates/compiler/mono/src/drop_specialization.rs

```diff
@@ -1490,3 +1498,60 @@ fn low_level_no_rc(lowlevel: &LowLevel) -> RC {
         }
     }
 }
+
+#[derive(Clone)]
+struct CountingMap<K>
+where
+    K: Eq + std::hash::Hash + Clone,
+{
+    map: MutMap<K, u64>,
+}
```

Comment:
would it work to not have a constraint on the type parameter here (but only on the impl bock below)? I'd prefer that

Reply:
I'll try